docs/ansible.md

```diff
@@ -187,3 +187,22 @@ For more information about Ansible and bastion hosts, read
 ## Mitogen
 
 You can use [mitogen](mitogen.md) to speed up kubespray.
+
+## Beyond ansible 2.9
+
+Ansible project has decided, in order to ease their maintenance burden, to split between
+two projects which are now joined under the Ansible umbrella.
+
+Ansible-base (2.10.x branch) will contain just the ansible language implementation while
+ansible modules that were previously bundled into a single repository will be part of the
+ansible 3.x package.Pleasee see [this blog post](https://blog.while-true-do.io/ansible-release-3-0-0/)
```

Comment:
Hmm, this will go again through the whole CI break dance but OK.

Reply:
Since we have to go through the CI again I took the opportunity to extend a bit more the docs and rearrange 2 commits.

@oomichi the bot removed the lgtm tag so when you have the time, please add it again.